frontend/src/components/newRequest/NewRequest.js

```diff
@@ -0,0 +1,212 @@
+import React, { useState } from "react";
+
+const NewRequestForm = ({ navigate }) => {
+  const [organisationName, setorganisationName] = useState("");
```

Comment:
Naming after 'set' shoulud be standardised- some are camel case and others are a mix

Reply:
Thanks this has now been amended